public/config.example.json

```
@@ -0,0 +1,95 @@
+{
+  "PUBLIC_URL": "http://example.com/",
+  "VITE_LOGO_URL": "https://example.com/logo.png",
+  "VITE_LOGO_ALT": "Alt description for my custom logo",
+  "VITE_DASHBOARD_BTN_URL": "https://dashboard.example.com",
+  "VITE_ANALYZE_BTN_URL": "https://analyze.example.com",
+  "VITE_SHOW_PUBLISH_BTN": false,
+  "VITE_STAC_API_URL": "https://api-endpoint.example.com",
+  "VITE_DEFAULT_COLLECTION": "collection-name",
+  "VITE_SCENE_TILER_URL": "https://titiler.example.com",
+  "VITE_SCENE_TILER_PARAMS": {
+    "sentinel-2-l2a": {
+      "assets": ["visual"]
+    },
+    "landsat-c2-l2": {
+      "assets": ["red", "green", "blue"],
+      "color_formula": "Gamma+RGB+1.7+Saturation+1.7+Sigmoidal+RGB+15+0.35"
+    },
+    "naip": {
+      "assets": ["image"],
+      "bidx": "1,2,3"
+    },
+    "cop-dem-glo-30": {
+      "assets": ["data"],
+      "colormap_name": "terrain",
+      "rescale": ["-1000,4000"]
+    },
+    "cop-dem-glo-90": {
+      "assets": ["data"],
+      "colormap_name": "terrain",
+      "rescale": ["-1000,4000"]
+    },
+    "sentinel-1-grd": {
+      "assets": ["vv"],
+      "rescale": ["0,250"],
+      "colormap_name": "plasma"
+    }
+  },
+  "VITE_MOSAIC_TILER_URL": "https://titiler-mosaic.example.com",
+  "VITE_MOSAIC_TILER_PARAMS": {
+    "sentinel-2-l2a": {
+      "assets": ["visual"]
+    },
+    "landsat-c2-l2": {
+      "assets": ["red"],
+      "color_formula": "Gamma+R+1.7+Sigmoidal+R+15+0.35"
+    },
+    "naip": {
+      "assets": ["image"],
+      "bidx": "1,2,3"
+    },
+    "cop-dem-glo-30": {
+      "assets": ["data"],
+      "colormap_name": "terrain",
+      "rescale": ["-1000,4000"]
+    },
+    "cop-dem-glo-90": {
+      "assets": ["data"],
+      "colormap_name": "terrain",
+      "rescale": ["-1000,4000"]
+    },
+    "sentinel-1-grd": {
+      "assets": ["vv"],
+      "rescale": ["0,250"],
+      "colormap_name": "plasma"
+    }
+  },
+  "VITE_MOSAIC_MIN_ZOOM_LEVEL": 7,
+  "VITE_SEARCH_MIN_ZOOM_LEVELS": {
+    "sentinel-2-l2a": {
+      "medium": 4,
+      "high": 7
+    },
+    "landsat-c2-l2": {
+      "medium": 4,
+      "high": 7
+    },
+    "naip": {
+      "medium": 10,
+      "high": 14
+    },
+    "cop-dem-glo-30": {
+      "medium": 6,
+      "high": 8
+    },
+    "cop-dem-glo-90": {
+      "medium": 6,
+      "high": 8
+    },
+    "sentinel-1-grd": {
+      "medium": 4,
```

```suggestion
      "medium": 7,
```

I just want to sneak this change into this PR